pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -726,8 +726,13 @@ async def publish_client(self, base_to_mirror_dir: str, pullspec, release_name,
         self.create_symlink(client_mirror_dir, False, False)
 
         # extract opm binaries
-        _, operator_registry = get_release_image_pullspec(pullspec, "operator-registry")
-        self.extract_opm(client_mirror_dir, release_name, operator_registry, build_arch)
+        if (major > 4 or minor >= 16):
```

Comment:
Can we instead test for the existence of this image, and extract when it does?

Reply:
when we add a new image in the payload it will always exist until it is deprecated, so I think there is no need to add an extra check for existence in each release.

Reply:
Yes, but we may have it in 4.17 while we do not in 4.16. It may or may not get backported to 4.15. Small change to not have us to worry about when to deploy this.

Reply:
we should have clear control of what to extract for what release, if in 4.16 the framework-tools pullspec is broken we should not fallback to operator-registry